pages/linux/resolvectl.md

```diff
@@ -0,0 +1,33 @@
+# resolvectl
+
+> Resolve domain names, IPV4 and IPv6 addresses, DNS resource records, and services.
+> Introspect and reconfigure the DNS resolver.
+> More information: <https://www.freedesktop.org/software/systemd/man/resolvectl.html>.
+
+- Show DNS settings:
+
+`resolvectl status`
+
+- Resolve the IPv4 and IPv6 addresses for one or more domains:
+
+`resolvectl query {{domain}}`
```

Comment:
Are domains space separated? Perhaps we could indicate this somehow?

Reply:
This way
```suggestion
`resolvectl query {{domain1 domain2 ...}}`
```

Reply:
Thanks @navarroaxel!